test_programs/compile_failure/signed_integer_literal_overflow/stderr.txt

```diff
@@ -0,0 +1,10 @@
+error: The value `SignedField { field: 128, is_negative: false }` cannot fit into `i8` which has range `-128..=127`
```

Comment:
Interesting, it seems `SignedField` doesn't have a good `to_string`, I'll try to fix this.